crates/cli-support/src/js/mod.rs

```diff
@@ -1248,6 +1239,12 @@ wasm = wasmInstance.exports;
         dst.push('}');
         ts_dst.push_str("}\n");
 
+        dst.push_str(&format!(
+            "
+                if (Symbol.dispose) {name}.prototype[Symbol.dispose] = {name}.prototype.free;
+            "
+        ));
```

Comment:
Why not put this inside the class definition with the help of `static { ... }`?

Reply:
These still only have 92% browser support (https://caniuse.com/mdn-javascript_classes_static_initialization_blocks).

We can do it, but it seems like dropping multiple percentages of browser support for no reason?

Reply:
Ah, good catch.